src/application.coffee

```diff
@@ -644,7 +653,28 @@ application.update = update = (force) ->
 			# Set the updating as finished in its own block, so it never has to worry about other code stopping this.
 			updateStatus.state = UPDATE_IDLE
 
+listenToEvents = ->
+	docker.getEventsAsync()
+	.then (stream) ->
+		parser = JSONStream.parse()
+		stream.pipe(parser)
+		parser.on 'data', (data) ->
+			if data?.Type == 'container'
```

Comment:
You could also check for `data.status in `[ 'die', 'restart' ]` to avoid needing to do the db query when it's not an event we care about.  In fact is it possible to specify to `docker events` that we only want certain types of events? That would be ideal